packages/node/src/index.ts

```diff
@@ -1,5 +1,208 @@
+          console.error('Error calling ReadMe API. Contact support@readme.io for help!');
+          console.error(e.data);
+        }
+        // Don't want to cause an error in their API
+        return next();
+      }
+    }
+
+    if (req.path === '/readme-webhook' && req.method === 'POST' && !options.disableWebhook) {
+      try {
+        verifyWebhook(req.body, req.headers['readme-signature'] as string, readmeProjectData.jwtSecret as string);
+        const user = await userFunction(req, getUser);
+        return res.send(user);
+      } catch (e) {
+        return res.status(400).json({ error: (e as Error).message });
+      }
+    } else if (req.path === '/readme-setup' && options.development) {
+      const setupHtml = buildSetupView({
+        baseUrl,
+        subdomain: readmeProjectData.subdomain as string,
+        stableVersion: readmeVersionData?.find(version => version.is_stable)?.version || '1.0',
+        readmeAPIKey,
+        disableMetrics: options.disableMetrics,
+        disableWebhook: options.disableWebhook,
+      });
+      return res.send(setupHtml);
+    } else if (req.path === '/webhook-test' && options.development) {
+      const email = req.query.email as string;
+      try {
+        const webhookData = await testVerifyWebhook(baseUrl, email, readmeProjectData.jwtSecret as string);
+        return res.json({ ...webhookData });
+      } catch (e) {
+        return res.status(400).json({ error: (e as Error).message });
+      }
+    }
+
+    try {
+      const user = await userFunction(req, getUser);
+      if (!user || !Object.keys(user).length || options.disableMetrics) return next();
+
+      const groupId = getGroupIdByApiKey(user, requestAPIKey);
+      if (!groupId) {
+        console.error(
+          usingManualAPIKey
+            ? 'The API key you passed in via `manualAPIKey` could not be found in the user object you provided.'
+            : 'Could not automatically find an API key in the request. You should pass the API key via `manualAPIKey` in the `getUser` function. Learn more here: https://docs.readme.com/main/docs/unified-snippet-docs#/getuserbyapikey',
+        );
+        return next();
+      }
+
+      log(
+        readmeAPIKey,
+        req,
+        res,
+        {
+          apiKey: groupId,
+          label: user.label ? user.label : user.name,
+          email: user.email,
+        },
+        options,
+      );
+    } catch (e) {
+      return next();
+    }
+    return next();
+  };
+};
+
+// I think this pattern will allow us to support both of these:
+// const readme = require('readmeio').auth('api-key');
+//
+// I don't like this as much but not sure how else we can do it
+// import { readme } from 'readmeio';
+// readme.auth('api-key');
+function auth(key: string) {
+  readmeAPIKey = key;
+  // Reset the cache for the ReadMe project if the api key changes
+  readmeProjectData = undefined;
+  return readme;
+}
+
+export { verifyWebhook, log, auth, getProjectBaseUrl, readme };
```

Comment:
I think we should leave the package name the same! I don't think it's worth a big change since we don't have a name we want to move to and we've been using `readmeio` forever. If we knew what we wanted I would be down, but I don't want to come up with something for the sake of it.

What about something like this for the export:

```
import { ReadMe } from 'readmeio';
const readme = ReadMe(key);
app.use(readme.express(() => {}));
```